docs/source/reference-overview/kibana-configuration.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Kibana configuration reference
+navigation_title: "Configuration reference"
```

Comment:
It's a minor thing for now, but for the future do we want to keep the term "reference" in the navigation title?